src/block-components/alignment/index.php

```diff
@@ -42,7 +42,7 @@
 	if ( ! function_exists( 'stackable_render_block_alignment_frontend_polyfill' ) ) {
 
 		function stackable_render_block_alignment_frontend_polyfill ( $block_content, $block ) {
-			if ( stripos( $block[ 'blockName' ], 'stackable/' ) === false ) {
+			if ( stripos( $block[ 'blockName' ] ?? '', 'stackable/' ) === false ) {
```

Comment:
We cannot use `??` because this is was introduced in PHP 7.0, we need to support lower versions of PHP